spec/features/inviting_new_submissions_in_place_of_expired_ones_feature_spec.rb

```diff
@@ -0,0 +1,40 @@
+# Testing rake tasks method found at:
+# http://carlosplusplus.github.io/blog/2014/02/01/testing-rake-tasks-with-rspec/
+# https://robots.thoughtbot.com/test-rake-tasks-like-a-boss
+
+require 'rails_helper'
+require 'rake'
+
+describe 'inviting new submissions in place of expired ones' do
```

Comment:
Well, the service's interface is `#call`, I don't think that this would be changed a lot or ever. And if somebody changes the interface or object name and doesn't grep for all the places he should change it at - he deserves all the errors he gets. Besides, I don't think you can ever test an app 100%, there will always be scenarios where something can fail and tests won't catch that. You should be able to live with that and say "good enough" sometimes :)